_overviews/scala3-book/domain-modeling-oop.md

```diff
@@ -74,14 +145,27 @@ trait GreetingService:
 trait TranslationService:
   def translate(text: String): String = "..."
 ```
+
+{% endtab %}
+{% endtabs %}
+
 To compose the two services, we can simply create a new trait extending them:
+
+{% tabs traits_5 class=tabs-scala-version %}
```

Comment:
`class=tabs-scala-version` should not be used for "Scala 2 and 3" tabs. Soon I am adding assertions to make CI fail for this
```suggestion
{% tabs traits_5 %}
```